data/blog/electron-3-0.md

```diff
@@ -0,0 +1,107 @@
+
+## Bug Fixes
+
+* [[#13397](https://github.com/electron/electron/pull/13397)] fix: issue with fs.statSyncNoException throwing exceptions
+* [[#13476](https://github.com/electron/electron/pull/13476), [#13452](https://github.com/electron/electron/pull/13452)] fix: crash when loading site with jquery
+* [[#14092](https://github.com/electron/electron/pull/14092)] fix: crash in net::ClientSocketHandle destructor
+* [[#14453](https://github.com/electron/electron/pull/14453)] fix: notify focus change right away rather not on next tick
+* [[#14205](https://github.com/electron/electron/pull/14205)] fix: pressing escape on a dialog now closes it even if defaultId is set
+
+### MacOS
+
+* [[#13220](https://github.com/electron/electron/pull/13220)] fix: issue allowing bundles to be selected in `<input file="type">` open file dialog
+* [[#12404](https://github.com/electron/electron/pull/12404)] fix: issue blocking main process when using async dialog
+* [[#12395](https://github.com/electron/electron/pull/12395)] fix: dark mode tray highlighting bug
+* [[#12043](https://github.com/electron/electron/pull/12043)] fix: context menu click callback
+* [[#12527](https://github.com/electron/electron/pull/12527)] fix: event leak on reuse of touchbar item
+* [[#12352](https://github.com/electron/electron/pull/12352)] fix: tray title crash
+* [[#12327](https://github.com/electron/electron/pull/12327)] fix: non-draggable regions
+* [[#12809](https://github.com/electron/electron/pull/12809)] fix: to prevent menu update while it's open
+* [[#13162](https://github.com/electron/electron/pull/13162)] fix: tray icon bounds not allowing negative values
+* [[#13085](https://github.com/electron/electron/pull/13085)] fix: tray title not inverting when highlighted
+* [[#12196](https://github.com/electron/electron/pull/12196)] fix: Mac build when `enable_run_as_node==false`
+* [[#12157](https://github.com/electron/electron/pull/12157)] fix: additional issues on frameless windows with vibrancy
+* [[#13326](https://github.com/electron/electron/pull/13326)] fix: to set mac protocol to none after calling `app.removeAsDefaultProtocolClient`
+* [[#13530](https://github.com/electron/electron/pull/13530)] fix: incorrect usage of private APIs in MAS build
+* [[#13517](https://github.com/electron/electron/pull/13517)] fix: `tray.setContextMen`u crash 
+
+### Linux
+
+* [[#12983](https://github.com/electron/electron/pull/12983)] fix: `Menu.setApplicationMenu(null)` crash
+* [[#12507](https://github.com/electron/electron/pull/12507)] fix: `BrowserWindow.focus()` for offscreen windows
+* [[#12192](https://github.com/electron/electron/pull/12192)] fix: use application name for notifications
+* [[#12388](https://github.com/electron/electron/pull/12388)] fix: use the appname as the tray icon's default tooltip
+
+### Windows
+
+# Security-Related Updates
+
+* [SECURITY] fix: webPreferences inheritance issue. CVE-2018-15685
```

Comment:
This is a little confusing: if I were reading this as a new user, I'd wonder if this meant that 2.0.x was still affected

Reply:
hmm i might not include this section tbh